src/gui/title_bar.cpp

```diff
@@ -0,0 +1,586 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2020-2023  The DOSBox Staging Team
+ *  Copyright (C) 2002-2021  The DOSBox Team
+ *
```

Comment:
(Probably just a copy-and-paste)

If this is a new for this PR, suggest `2023-2023  The DOSBox Staging Team` only; the copyright lines are just facts about who authored the file, when it was created, and when it was last modified.

(same w/ the header file)

Reply:
OK, changed.